static/src/javascripts/projects/commercial/modules/prebid/bid-config.js

```diff
@@ -349,11 +353,18 @@ const isPbTestOn = (): boolean => !isEmpty(pbTestNameMap());
 const inPbTestOr = (liveClause: boolean): boolean => isPbTestOn() || liveClause;
 
 /* Bidders */
-const appNexusBidder: PrebidBidder = {
-    name: 'and',
+const appNexusBidderUkRow: PrebidBidder = {
+    name: 'and-uk-row',
+    switchName: 'prebidAppnexus',
+    bidParams: (slotId: string, sizes: PrebidSize[]): PrebidAppNexusParams =>
+        getAppNexusBidParamsUkRow(sizes),
+};
+
+const appNexusBidderAu: PrebidBidder = {
+    name: 'and-au',
```

Comment:
The name eventually gets passed to Prebid as the `bidder` value: 
```
bidder | String | The bidder code. Used by ad server’s line items to identify bidders
```
We currently have an alias set up in Prebid for `and` so this will probably need updating. Does `and-uk-row` need to be added as an alias?

https://github.com/guardian/Prebid.js/blob/master/modules/appnexusBidAdapter.js#L33

Reply:
🤔 Do we need to make these different names? If the only reason is for different PBtest params, one option might be to use the switch name instead?